### PR TITLE
Updating zkSync network details

### PR DESCRIPTION
**Pull request by githubdoramon**
- updates RPC urls for mainnet
- update network names - (https://blog.matter-labs.io/8b8964ba7c59)
- adds logo

### CODE REVIEW

File: _data/icons/zksync-era.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://Qma6H9xd8Ydah1bAFnmDuau1jeMh5NjGEL8tpdnjLbJ7m2",
+    "width": 512,
+    "height": 512,
```
Comment (ligi) [inline _data/icons/zksync-era.json:5]: identify Qma6H9xd8Ydah1bAFnmDuau1jeMh5NjGEL8tpdnjLbJ7m2         16:25:34
Qma6H9xd8Ydah1bAFnmDuau1jeMh5NjGEL8tpdnjLbJ7m2 SVG 517x517 517x517+0+0 8-bit sRGB 2308B 0.000u 0:00.009

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/zksync-era.json (hunk 1 above)
Comment (githubdoramon) [reply _data/icons/zksync-era.json:5]: This is quite weird.. accessing it I get the following
image

Comment (ligi) [reply _data/icons/zksync-era.json:5]: hm - weird - can you upload to iconDownload folder? In the end it needs to pass CI

Comment (githubdoramon) [reply _data/icons/zksync-era.json:5]: added

**Review by ligi — APPROVED** [review]
Interesting - seemed to be a fluke of imagemagic

File: _data/icons/zksync-era.json (hunk 1 above)
Comment (ETHeOReDrheTorICaL) [reply _data/icons/zksync-era.json:5]: How do I use this to update MetaMask ......   create a repository first ????

**Pull request merged**